src/main/java/org/cryptomator/cryptofs/CryptoFileSystems.java

```diff
@@ -54,21 +54,41 @@ public CryptoFileSystemImpl create(CryptoFileSystemProvider provider, Path pathT
 			rawKey = key.getEncoded();
 			var config = configLoader.verify(rawKey, Constants.VAULT_VERSION);
 			var adjustedProperties = adjustForCapabilities(pathToVault, properties);
-			return fileSystems.compute(normalizedPathToVault, (path, fs) -> {
-				if (fs == null) {
-					return create(provider, normalizedPathToVault, adjustedProperties, key, config);
-				} else {
-					throw new FileSystemAlreadyExistsException();
-				}
-			});
+			var cryptor = config.getCipherCombo().getCryptorProvider(csprng).withKey(key);
```

Comment:
Until there is new API related to cryptomator/cryptolib#19, make a copy, so the destruction of `key` at the end of this method doesn't void the cryptor.

```suggestion
			var keyCopy = Masterkey.createFromRaw(key.getEncoded()); // TODO replace with key.clone() eventually (depends on cryptomator/cryptolib#19
			var cryptor = config.getCipherCombo().getCryptorProvider(csprng).withKey(keyCopy); 

```